packages/sanity/src/desk/panes/document/DocumentPaneProvider.tsx

```diff
@@ -450,10 +456,13 @@ export const DocumentPaneProvider = memo((props: DocumentPaneProviderProps) => {
 
   // Reset `focusPath` when `documentId` or `params.path` changes
   useEffect(() => {
-    // Reset focus path
-    setFocusPath(params.path ? pathFromString(params.path) : [])
-    onSetOpenPath([])
-  }, [params.path, documentId])
+    // Reset focus path when url params path changes
+    const pathFromUrl = params.path ? pathFromString(params.path) : []
+    setFocusPath(pathFromUrl)
+    // @todo refactor this, this doesn't set the openPath if it has indexSegment
+    const hasIndexPath = pathFromUrl.some((segment) => isIndexSegment(segment))
```

Comment:
@bjoerge could you look at this if this makes sense? If there are index paths which there will be for PTE this ensures studio doesn't crash where the openpath has indexes instead of keys